Stage1/common/create_gif.py

```diff
@@ -0,0 +1,17 @@
+import os
+import imageio
+import natsort  # for actually sort file
+
+def create_gif(png_dir='../Media/png', fps=24):
+    images = []
+    for file_name in natsort.natsorted(os.listdir(png_dir)):
+        if file_name.endswith('.png'):
+            file_path = os.path.join(png_dir, file_name)
+            try:
+                images.append(imageio.imread(file_path))
+            except ValueError:
+                pass
+    imageio.mimsave(os.path.join(png_dir, 'movie.gif'), images, fps=fps)
```

Comment:
Consider adding some variability in the name so new result doesn't necessary override last one. 
Probably include it as a parameter for function.